linera-core/src/worker.rs

```diff
@@ -903,7 +903,7 @@ where
             owner,
             signature,
             blobs,
-            validated: _,
+            validated,
```

Comment:
nit: (not from this PR but) How about `validation_certificate`? `validated` reads like a boolean value.

Reply:
Good point. But yes, I'd make that a separate PR.